### ISSUE TITLE
Can you add an option to disable switching guns

### ISSUE THREAD

**Issue by alexvornoffice** [PowerOfUniverse/Surviv.io-Cheats#15]
A lot of times recently the guns are switching automatically to pistols? I have ak and it pick-up pistol in a fight!? very frustrating...

**Comment by alexvornoffice** [comment]
Add option never to pick up guns.

**Comment by alexvornoffice** [comment]
I wanted to fix myself, and to change the code but now I see the all code is "minified" in a line, it is possible to get clean code?

**Comment by PowerOfUniverse** [comment]
The problem lies in the speed of refresh, I will fix it in my free time.

**Comment by alexvornoffice** [comment]
I unimified the code with one site unminify.com/ and edited the init.js file and it seems to help.

**Comment by PowerOfUniverse** [comment]
You can always paste the code here, I will replace it.

**Comment by IME9** [comment]
> Add option to never to pick up guns.

This thing already exist look

screenshot : https://gyazo.com/1f960587076f168f9015a7d08b920932

**Comment by alexvornoffice** [comment]
yeah, but I want to pick up ammo, armor, helmet, pills, soda, but NO guns, now I need to press F button 100 times per game..

**Comment by alexvornoffice** [comment]
sometimes opening doors may trigger pick up guns we don't want, so yes maybe the plugin wants to pick up ammo, or something else, but when we move - it picks up other guns or pistols, maybe this is the problem...
